components/pages/Dashboard/Dialogs/DeleteAllFeedTimesDialog.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Dialog, Classes } from '@blueprintjs/core'
+import { Button } from '../../../primitives'
+
+const _ = ({ isOpen, close }) => {
+  /* TODO:  Add api interaction */
+
+  return (
+    <Dialog
+      icon="trash"
+      onClose={() => close()}
+      title="Delete All Feed Times"
+      isOpen={isOpen}
+    >
+      <>
+        <div className="flex flex-col items-center m-auto p-4">
+          {/* Wrapped in another div to keep span element inline */}
+          <div>Are you sure you want to delete <span className="font-bold">all</span> scheduled feed times? This action cannot be undone!</div>
```

Comment:
This is a really tiny nitpick, but can you use the `Typography` primitive for all typography variance (size, weight, etc) just for consistency? Here it would look like
```
<Typography variant="body" weight="bold">all</Typography>
```
...and you would add it to the list of imports from `../../../primitives` at the top. Otherwise it looks great!

Reply:
Update submitted